code/form/style.css

```diff
@@ -0,0 +1,116 @@
+/*** Header ***/
+
+header {
+  background-color: white;
+  height: 80px;
+  display: flex;
+  position: sticky;
+  top: 0;
+  z-index: 100;
+  align-items: center;
+  justify-content: space-between;
+  overflow: hidden;
+  padding: 0 16px;
+  opacity: 0.98;
+}
+
+header img {
+  width: 180px;
+}
+
+img[src="./img/user_icon_flaticon.png"] {

```

Comment:
Using a class would probably be smoother here :) Using the brackets is a really nice feature though!